src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj

```diff
@@ -107,8 +107,8 @@
     <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="8.6.2" />
     <PackageReference Include="Reactive.Annex.Uno" Version="0.2.0-dev.14" />
     <PackageReference Include="GeneratedSerializers.Json" Version="0.2.0-dev.10" />
-    <PackageReference Include="Uno.UI" Version="3.1.10" />
-    <PackageReference Include="Uno.UI.RemoteControl" Version="3.1.10" />
+    <PackageReference Include="Uno.UI" Version="3.4.0" />
```

Comment:
Let's stay on 3.1.10 for now, that's the version we have "tested".

Reply:
After discussion we can go for 3.4

Reply:
As discussed with you, we're going to keep it to 3.4.0 as I saw this bug occurring on Android with the 3.1.10 Uno version. For context purposes, this is the bug I've found on the chucknorris page.

![scre](https://user-images.githubusercontent.com/17461593/104062364-57022700-51c8-11eb-9446-2a0af4e35675.jpg)